ngi_pipeline/engines/piper_ngi/local_process_tracking.py

```diff
@@ -73,12 +98,7 @@ def update_charon_with_local_jobs_status(quiet=False, config=None, config_file_p
                                                                    e))
                 LOG.error(error_text)
                 if not config.get('quiet'):
-                    mail_analysis(project_name=project_name,
-                                  sample_name=sample_id,
-                                  engine_name=engine,
-                                  level="ERROR",
-                                  info_text=error_text,
-                                  workflow=workflow)
+                    mail_ngi_log_message(sample_env,"ERROR",error_text)
```

Comment:
If you were to merge `mail_ngi_log_message`and `mail_analysis` by adding a parameter expecting a SampleEnv to mail_analysis it would probably be easier to maintain.